docker-compose.yaml

```diff
@@ -0,0 +1,72 @@
+version: '3.9'
+
+services:
+    php:
+        container_name: foundry_php
+        build:
+            context: .
+            dockerfile: ./docker/Dockerfile
+        depends_on:
+            mysql:
+                condition: service_healthy
+            postgres:
+                condition: service_healthy
+            mongo:
+                condition: service_healthy
+        volumes:
+            - .:/app
+        working_dir: /app
+
+    mysql:
+        container_name: foundry_mysql
+        image: mysql
+        command: --default-authentication-plugin=mysql_native_password
+        environment:
+            MYSQL_ROOT_PASSWORD: root
+            MYSQL_DATABASE: zenstruck_foundry
+        ports:
+            - "3306:3306"
```

Comment:
Should we use a non-standard local port? (same with postgres/mongo below)

Reply:
I'd like to provide the way to use a .env and to let the user chose. We can actually say `${MYSQL_PORT:-3306}`

Reply:
Interesting, I'm not aware of that syntax, does that mean "use MYSQL_PORT if defined, otherwise use 3306"?

Reply:
absolutely. This syntax comes from bash